src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -315,6 +315,28 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, new_account_details);
         });
 
+        it('cli account update name undefined (and back to original name)', async function() {
```

Comment:
@naveenpaul1 sure, I can add it.
But I want to understand that there is no difference because the command that we are running using template literals, then adding a value of undefined to it would cause adding a string of undefined.